WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -379,3 +379,25 @@ $(function() {
     $(this).text(formatted);
   });
 });
+
+// Handler for locale changes.
+$(function() {
+  $('#locale-selector').on('click', 'a', function(e) {
+    e.preventDefault();
+    e.stopPropagation();
+
+    // More or less copied from
+    // https://github.com/rails/jquery-ujs/blob/9e805c90c8cfc57b39967052e1e9013ccb318cf8/src/rails.js#L215.
+    var href = this.href;
+    var method = "patch";
+    var csrfToken = $('meta[name=csrf-token]').attr('content');
+    var csrfParam = $('meta[name=csrf-param]').attr('content');
+    var form = $('<form method="post" action="' + href + '"></form>');
```

Comment:
`href` is used only here, so you can maybe just use `this.href` here and drop the variable.

Reply:
Good idea! You are seeing evidence of my copy paste-ing =)